workflow-templates/phpunit-oci.yml

```diff
@@ -68,28 +68,27 @@ jobs:
       matrix:
         php-versions: ${{ fromJson(needs.matrix.outputs.php-version) }}
         server-versions: ${{ fromJson(needs.matrix.outputs.server-max) }}
+        oci-versions: ['11', '18', '21', '23']
```

Comment:
while this works for talk, it does not work for all apps using this action template.
21+ support is only possible with Nextcloud 30+

@icewind1991 can we somehow in corporate this into your action?